package.json

```diff
@@ -1,102 +1,94 @@
 {
-    "name": "skeleton",
-    "label": "Skeleton Package",
+    "name": "zoho",
+    "label": "Zoho",
     "dependencies": [
         {
             "type": "service",
             "name": "http",
-            "version": "v1.6.3",
+            "version": "v1.7.0",
             "instanceId": "shared"
         },
         {
             "type": "package",
             "name": "oauth",
-            "version": "v1.0.24"
+            "version": "v1.0.31"
```

Comment:
@sebasmoyano 
I agree, the stable fixed version of oauth still has to be set in the package.json but I can do without the readme version (in this case it is better not to keep it there).